docs/UserGuide.md

```diff
@@ -448,38 +471,38 @@ Expected output:
 
 ### Marking a delivery : `mark -d`
 
-Marks the specified delivery in VendorVault with the specified `STATUS`.
+Marks the specified delivery with the specified status in VendorVault.
 
 Format: `mark -d INDEX STATUS`
 
-<box type="details" seamless>
+<box type="info" seamless>
 
 Parameters:
-
 - `INDEX`: The index of the delivery to be marked in the displayed list. It must be a number between 1 and the total number of deliveries displayed (inclusive), and cannot be blank.
-- `STATUS`: The status of delivery. It must be one of the following values: `PENDING`, `DELIVERED`, `CANCELLED`, and cannot be blank.
-
+- `STATUS`: The delivery status to change to. It must be one of the following values: `PENDING`, `DELIVERED`, `CANCELLED`, and cannot be blank.
 
 </box>
 
-<box type="warning" seamless>
+<box type="caution" seamless>
 
 **Warnings**:
 - At least one space is needed between `mark` and `-d`.
 - At least one space is needed between `-d` and `INDEX`.
-- At least one space is needed between `INDEX` and `STATUS.
+- At least one space is needed between `INDEX` and `STATUS`.
+- All parameters must be given.
```

Comment:
there is only one parameter for this command which is `INDEX` so this may not be needed